### ISSUE TITLE
List of operators for regionless locations

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#1036]
Currently, we don't have a way to add operators to regionless locations. In the meantime, here's a list:

Location: Work-Work
Games are operated by Pop Bumper Trondheim, http://popbumper.no/

**Comment by scottwainstock** [comment]
@RyanTG Why can't we add operators to regionless locations?

**Comment by scottwainstock** [comment]
<img width="717" alt="screen shot 2018-08-21 at 8 00 24 pm" src="https://user-images.githubusercontent.com/179757/44440441-efa69680-a57c-11e8-8031-1a3e00b706ac.png">

**Comment by RyanTG** [comment]
Because those operators are tied to regions? I don't know. But you can't.

Steps:
- go to regionless, and lookup a regionless location.
- click the edit location button.
- click the operator dropdown.

Result: the dropdown has no options.

Can you tell me the steps you took? Guessing you did it via rails admin?

**Comment by scottwainstock** [comment]
I did use rails admin, yeah. So the issue here is that non-admins can't do it?

**Comment by RyanTG** [comment]
Yes, I guess it is! Usually that's a user-editable field.

But part and parcel to this issue is that I'm pretty sure operators _have_ to be tied to a region. So, if the backroads of Italy are not a Region, but there's an operator who only operates in that area, I can't add that fellow as a "regionless operator" and then tag him at that location.

**Comment by RyanTG** [comment]
I've been wondering: Should we make _all_ operators global? Some might be in two regions already (for example: chicago & indiana - just guessing here), and some might be both regionless and in a region.

I'm guessing they all have unique names.

**Comment by scottwainstock** [comment]
@RyanTG I agree that they SHOULD all be global, but I think we'll have an issue with the old apps if we do it that way. The old apps only pull down the list of operators for their region. So, we might have to think of a phased way to add (remove) this.

**Comment by scottwainstock** [comment]
On a region'ed page: operator dropdown should be regional + regionless
On a regionless page: ALL operators show in dropdown (regioned and regionless)

**Comment by scottwainstock** [comment]
https://github.com/scottwainstock/pbm/commit/73c909765b40613bcf2867a38ffbef721717b4e1

**Comment by RyanTG** [comment]
```
On a region'ed page: operator dropdown should be regional + regionless
On a regionless page: ALL operators show in dropdown (regioned and regionless)
````

Sounds to me like both regional locations and regionless locations show ALL operators. And looking at the map, it looks like that, too. Am I missing something?

**Comment by scottwainstock** [comment]
Can I get some links to examples?

**Comment by RyanTG** [comment]
Region location: https://pinballmap.com/youngstown/?by_location_id=8208
Regionless location: https://pinballmap.com/regionless/?by_location_id=10798

Hmmm, maybe I'm seeing this because I'm logged in as a super user? But like I said, it sounds like you meant to have it like this, unless I'm misreading your description.

**Comment by RyanTG** [comment]
Ah ok. Part of the problem was that I was misreading your comment.

Should be:

````
On a region'ed page: operator dropdown should be THAT region + regionless
On a regionless page: ALL operators show in dropdown (regioned and regionless)
````

Next up! How do I add a regionless operator? With super admin, if I add an operator and choose "no region" in the dropdown, the operator still gets added to Los Angeles (and if I choose a region, they get added to Los Angeles). See this issue that I closed for no reason https://github.com/scottwainstock/pbm/issues/480

**Comment by scottwainstock** [comment]
I am seeing these problems... weird.

**Comment by scottwainstock** [comment]
This should do it: https://github.com/scottwainstock/pbm/commit/2933960faad28b0efbf4a8bd46a22d3f0727855b
